apps/docs/src/app/core/component-docs/table/examples/table-column-sorting-example.component.html

```diff
@@ -0,0 +1,51 @@
+<table fd-table>
+    <thead fd-table-header>
+        <tr fd-table-row>
+            <th fd-table-cell>
+                <fd-popover fd-table-popover [(isOpen)]="open">
```

Comment:
I don't think that makes sense here because you would have to use a button with [fdMenu] inputs etc.  And we don't want a button, we want the whole header cell to handle the click